contract/info.md

```diff
@@ -0,0 +1,8 @@
+### Shibuya
+* 0xF183f51D3E8dfb2513c15B046F848D4a68bd3F5D
+
+### Shiden
+* [0x20044438CfaF684e251d1FfC70f999291D49e9a7](https://blockscout.com/shiden/address/0x20044438CfaF684e251d1FfC70f999291D49e9a7/transactions)
+
+### Astar
```

Comment:
there is a file in the root that covers all deployment addresses
https://github.com/AstarNetwork/astarbase/blob/main/contract/deployment-info.md